<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataUnitViewer</class>
 <widget class="QMainWindow" name="DataUnitViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataElementViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTableWidget" name="outputTable">
      <column>
       <property name="text">
        <string>rawData</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>s/c</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>dir</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>prot</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>channel</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>subType</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>sender</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>receiver</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>valid type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>message</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>valid message</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>host</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="filterIn">
          <property name="text">
           <string>In</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="filterOut">
          <property name="text">
           <string>Out</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="filterServer">
          <property name="text">
           <string>Server</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="filterClient">
          <property name="text">
           <string>Client</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="filterTcp">
          <property name="text">
           <string>TCP</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="filterUdpUnicast">
          <property name="text">
           <string>UDP Unicast</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="filterUdpBroadcast">
          <property name="text">
           <string>UDP Broadcast</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Channel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="filterChannel"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="filterType"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>SubType</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="filterSubType"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget3">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Sender</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="filterSender"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Receiver</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="filterReceiver"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Host</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="filterHost"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget4">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="filterNullMessage">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Always hide null messages</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="filterInvalidType">
          <property name="text">
           <string>Always show invalid type</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="filterInvalidMessage">
          <property name="text">
           <string>Always show invalid message</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
